src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -929,7 +929,7 @@
       "courseId": "idOfTypicalCourse1.access",
       "feedbackQuestionId": "3",
       "feedbackQuestionType": "TEXT",
-      "giver": "instructor1@course1.tmt",
+      "giver": "student1InCourse1.access@gmail.tmt",
```

Comment:
The question of this response does not have feedback path `students` -> `general`? It is `self` -> `general`. So I guess you should not change this.

Reply:
`self` means the creator of the feedback session (which is an instructor).

> the `isCommentFromFeedbackParticipant ` on line 1061 is `false`

This is valid. I can give comment as an instructor to a response (it is just that the response is given by myself). I understand that you are trying to make the test data more reasonable. If the change is small, go ahead.